lib/api/controller/security.js

```diff
@@ -1159,13 +1165,25 @@ class SecurityController extends NativeController {
    * @returns {Promise}
    * @private
    */
-  async _persistUser(request, profileIds, content) {
-    const id = this.getId(request, ifMissingEnum.GENERATE);
+  async _persistUser(request, profileIds, content, humanReadableId=true) {
```

Comment:
```suggestion
  async _persistUser(request, profileIds, content, { humanReadableId=true } = {}) {
```

Options should be passed as an object. This improve readability:
```
function foobar (index, collection, skipRights=false) { ... }
function barfoo (index, collection, { skipRights=false } = {}) { ... }

foobar('idx', 'col', true); // what does true mean ??
barfoo('idx', 'col', { skipRights: true }); // explicite
```